view_index.php

```diff
@@ -0,0 +1,39 @@
+<?php
+require_once('controller.php');
+
+$todos = new Controller('post_todo');
+$todos->index();
```

Comment:
26行目でも同じ処理がはしっているので、こちらはreturn値もあることですし変数に置いてあげて、その変数を呼び出すのがいいかと思います。

```suggestion
$list = $todos->index();
```

処理が2回はしるとその分の処理時間・負担がかかるので、なるべく避けて書くのベストです。